src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/editor/labeler/marker/Marker.kt

```diff
@@ -313,12 +313,40 @@ private fun FieldBorderCanvas(
                             state.getEntryIndexesByBorderIndex(pointIndex).second == entryIndex
                         ) 1f else IDLE_LINE_ALPHA
                     if (border in screenRange) {
+                        var strokeWidth: Float = STROKE_WIDTH
+                        var color = borderColor.copy(alpha = borderLineAlpha)
+                        if (editorConf.highlightCurrentEntryBorder) {
+                            val currentIndex = editorState.project.currentModule.currentIndex
+                            if (currentIndex == entryIndex || currentIndex == entryIndex - 1) {
+                                strokeWidth = editorConf.currentEntryBorderHighlightWidth
+                                color = editorConf.currentEntryBorderHighlightColor.toColor()
+                            }
+                        }
+                        // when dragging, the highlight border would glitch, but I have no idea how to fix that
+                        if (editorConf.highlightCursorPositionEntryBorder) {
```

Comment:
The most simple fix:
```suggestion
                        if (editorConf.highlightCursorPositionEntryBorder &&
                            cursorState.value.mouse != MarkerCursorState.Mouse.Dragging
                        ) {
```

But this will cause the highlight to disappear when dragging.

So if we want it perfectly, we need to add a `previousPositionBeforeDragging` nullable field in `MarkerCursorState`:

1. in `startDragging`, copy the `position` value to `previousPositionBeforeDragging`
2. during dragging, do not change it
3. in `finishDragging`, clear it
4. when drawing the highlight, get the position according to the `mouse` type. If it's dragging, use `previousPositionBeforeDragging`, otherwise, use `position`

Reply:
Um, I think I was wrong.
Even if we remember the cursor position before we start dragging, the borders are changing, causing the entry selected also changing.

So we probably should instead remember a relative offset of the clicked position and the parameter line being dragged, so when rendering the lines, we can get a stable cursor position.

Reply:
I got it working. Please have a look!

[patch.diff.zip](https://github.com/user-attachments/files/17649204/patch.diff.zip)